tests/CMakeLists.txt

```diff
@@ -26,6 +26,7 @@ set(SOURCES
     util/event_loop.cpp
     util/test_file.cpp
     util/test_utils.cpp
+    util/bson.cpp
```

Comment:
Keep the list of files sorted. This new test file doesn't go in `util`; that directory is utilities for the tests, not tests for utilities.